contributors/devel/sig-architecture/conformance-tests.md

```diff
@@ -181,6 +181,57 @@ configured. Conformance tests must be able to be run against clusters that have
 not been created with `test-infra/kubetest`, just provide a kubeconfig with the
 appropriate endpoint and credentials.
 
+### Running Conformance Tests With [KinD](https://kind.sigs.k8s.io/)
+
+1. Work in your kubernetes branch, preferably in the default go src location
+
+2. Create your kind node image:
+
+```sh
+kind build node-image --type bazel
```

Comment:
I use bazel because if its caching capabilities, but I've checked that some users struggle with it and prefer to use Make.
KIND allows to use current kubernetes/kubernetes Makefiles , maybe we can mention this or add the step necessary for both?

Reply:
Sure!